### DISCUSSION TITLE
"Connecting to Databases/index.mdx": no built-in support for Postgres

### DISCUSSION THREAD

**Discussion by frafra** [apache/superset#19214] [Q&A / Help]
https://github.com/apache/superset/blob/f2b802978d99b0cf329520c120b2decd5953ddd0/docs/src/pages/docs/Connecting%20to%20Databases/index.mdx#L19

https://github.com/apache/superset/blob/f2b802978d99b0cf329520c120b2decd5953ddd0/docs/src/pages/docs/Connecting%20to%20Databases/postgres.mdx#L11-L12

https://github.com/apache/superset/blob/f2b802978d99b0cf329520c120b2decd5953ddd0/requirements/docker.in#L19

**Comment by BIT-9933** [comment]
Gy

**Comment by frafra** [comment]
```
for pkg in psycopg2 mysql presto; do grep -r $pkg requirements/; done
requirements/development.txt:psycopg2-binary==2.8.5
requirements/docker.txt:psycopg2-binary==2.9.1
requirements/development.txt:mysqlclient==1.4.2.post1
requirements/testing.txt:pyhive[hive,presto]==0.6.4
```

I think the documentation could add that development and Docker versions have built-in support for some databases or something like that.
